src/core/zowe/core_for_zowe_sdk/config_file.py

```diff
@@ -116,7 +118,12 @@ def location(self, dirname: str) -> None:
         else:
             raise FileNotFoundError(f"given path {dirname} is not valid")
 
-    def init_from_file(self) -> None:
+    def init_from_file(
+        self, 
+        config_type: str, 
+        validate_schema: Optional[bool] = False,
```

Comment:
I think the default value for `validate_schema` should be True 😋

Reply:
Setting this to True, cause failing some unit tests. That's why I set it to False. 
I will make some changes to those tests :+1: